charts/redpanda/values.go

```diff
@@ -168,10 +169,11 @@ type Logging struct {
 }
 
 type Monitoring struct {
-	Enabled        *bool             `json:"enabled" jsonschema:"required"`
-	ScrapeInterval string            `json:"scrapeInterval" jsonschema:"required,pattern=.*[smh]$"`
-	Labels         map[string]string `json:"labels"`
-	TLSConfig      map[string]any    `json:"tlsConfig"`
+	Enabled        *bool                   `json:"enabled" jsonschema:"required"`
+	ScrapeInterval monitoringv1.Duration   `json:"scrapeInterval" jsonschema:"required"`
```

Comment:
I think we'll want to add a special case to `genschema` via the `Mapper` function to add the pattern back to the JSON schema.